src/BPEssentials/Commands/Moderation/Jail.cs

```diff
@@ -0,0 +1,39 @@
+﻿using BPEssentials.Abstractions;
+using BPEssentials.ExtensionMethods;
+using BrokeProtocol.Entities;
+using BrokeProtocol.Required;
+using BrokeProtocol.Utility;
+using BrokeProtocol.Utility.Jobs;
+using BrokeProtocol.Utility.Networking;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace BPEssentials.Commands
+{
+    public class Jail : Command
+    {
+        public void Invoke(ShPlayer player, ShPlayer target, float timeInSeconds)
+		{
+			SvPlayer svplayer = player.svPlayer;
+			SvMovable svMovable = player.svMovable;
```

Comment:
```suggestion
```
Can we get rid of this, instead of `svMovable`, `svPlayer` wich in this case would be `target` should work too.